example/src/App.tsx

```diff
@@ -41,6 +46,10 @@ const App = () => {
   const [isProgrammaticBannerCreated, setIsProgrammaticBannerCreated] = useState(false);
   const [isProgrammaticBannerShowing, setIsProgrammaticBannerShowing] = useState(false);
   const [isNativeUIBannerShowing, setIsNativeUIBannerShowing] = useState(false);
+  const [isProgrammaticMRecCreated, setIsProgrammaticMRecCreated] = useState(false);
+  const [isProgrammaticMRecShowing, setIsProgrammaticMRecShowing] = useState(false);
+  const [isNativeUIMRecCreated, setIsNativeUIMRecCreated] = useState(false);
```

Comment:
Are you sure we need this state? Native UI banners only use one state to track `setIsNativeUIBannerShowing`.

Reply:
If programmatic MREC is tried to load when UI MRec is hidden, an error is thrown that an ad refresh is already in progress. Hence using `isNativeUIMRecCreated` to disable programmatic MRec button when UI MRec is shown or hidden.

Reply:
I don't follow. what's the error specifically? Banners and MRECs should not have different behaviorus.

Reply:
Here's the error: 
2021-05-06 17:17:29.233165-0700 AppLovinMAX Example[36506:4259108] [AppLovinSdk] ERROR [MAAdView] Unable to load a new ad. An ad refresh has already been scheduled in 22.23 seconds.

Reply:
Banners / MRECs should have same behaviour - can we figure out why this is different for you?

Reply:
Banners behaviour is the same. 

In one session, we can either show UI or programmatic, not both. 

Removing `isNativeUIMRecCreated` to match with banners.